core/src/main/scala/gospeak/core/services/youtube/YoutubeConf.scala

```diff
@@ -0,0 +1,3 @@
+package gospeak.core.services.youtube
+
+final case class YoutubeConf(secret: String)
```

Comment:
```suggestion
final case class YoutubeConf(secret: Secret)
```
(So value will not leak into logs...)